packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -0,0 +1,165 @@
+import { inspection } from '@chainlink/gauntlet-core/dist/utils'
+import { getRDD } from '../../../../lib/rdd'
+import { InspectInstruction, InspectionInput, instructionToInspectCommand } from '../../../abstract/inspectionWrapper'
+import { getOffchainConfigInput, OffchainConfig } from '../setConfig'
+
+type Expected = {
+  description: string
+  decimals: string | number
+  minAnswer: string | number
+  maxAnswer: string | number
+  transmitters: string[]
+  billingAccessController: string
+  requesterAccessController: string
+  link: string
+  billing: {
+    observationPaymentGjuels: string
+    recommendedGasPrice: string
+    transmissionPaymentGjuels: string
+  }
+  offchainConfig: OffchainConfig
+}
+
+const makeInput = async (flags: any, args: string[]): Promise<InspectionInput<any, Expected>> => {
+  if (flags.input) return flags.input as InspectionInput<any, Expected>
+  const rdd = getRDD(flags.rdd)
+  const contract = args[0]
+  const info = rdd.contracts[contract]
+  const aggregatorOperators: string[] = info.oracles.map((o) => o.operator)
+  const transmitters = aggregatorOperators.map((operator) => rdd.operators[operator].ocrNodeAddress[0])
+  const billingAccessController = flags.billingAccessController || process.env.BILLING_ACCESS_CONTROLLER
+  const requesterAccessController = flags.requesterAccessController || process.env.REQUESTER_ACCESS_CONTROLLER
+  const link = flags.link || process.env.LINK
+  const offchainConfig = getOffchainConfigInput(rdd, contract)
+  return {
+    expected: {
+      description: info.name,
+      decimals: info.decimals,
+      minAnswer: info.minSubmissionValue,
+      maxAnswer: info.maxSubmissionValue,
+      transmitters,
+      billingAccessController,
+      requesterAccessController,
+      link,
+      offchainConfig,
+      billing: {
+        observationPaymentGjuels: info.billing.observationPaymentGjuels,
+        recommendedGasPrice: info.billing.recommendedGasPrice,
+        transmissionPaymentGjuels: info.billing.transmissionPaymentGjuels,
+      },
+    },
+  }
+}
+
+const makeOnchainData = (instructionsData: any[]): Expected => {
```

Comment:
It would be more readable (also safer when typed) to name instruction outputs in an object instead of appending them to the array. Please consider for future improvements.

Reply:
I'll open an issue for this, currently no time to do it

Reply:
#86